### PR TITLE
fixed readme

### PR DESCRIPTION
**Pull request by HarrisHasnain**

### CODE REVIEW

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

File: Dockerfile [hunk 1]
```diff
@@ -23,21 +23,28 @@ RUN apt-get -yq install --no-install-recommends \
         tcpdump traceroute iproute2 dnsutils whois mtr iftop iputils-ping wget nmap netcat-traditional \
         procps \
         htop \
-    # Clean up
-    && apt-get autoremove -y \
-    && apt-get clean -y \
-    && rm -rf /var/lib/apt/lists/* 
+        screen tmux \
```
Comment (growlf) [inline Dockerfile:26]: Good additions! Bravo!

File: docker-compose.yml [hunk 2]
```diff
@@ -8,22 +8,22 @@ services:
...
     tty: true
     command: ["/bin/zsh"]
     volumes:
-      - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
-      - ${HOME}/.ssh:/home/ubuntu/.ssh
-      - ${PWD}:/app
-      - /var/run/docker.sock:/var/run/docker.sock
+    - ${HOME}/.gitconfig:/home/ubuntu/.gitconfig
+    - ${HOME}/.ssh:/home/ubuntu/.ssh
+    - .:/src
```
Comment (growlf) [inline docker-compose.yml:18]: Generally better practice to use the shell variable and not globbing which might not act as expected.

File: src/requirements.txt [hunk 3]
```diff
@@ -6,3 +6,4 @@ portscan==1.1
 pyfiglet==1.0.2
 speedtest-cli==2.1.3
 urllib3==2.1.0
+ansible==9.2.0
```
Comment (growlf) [inline src/requirements.txt:9]: Good catch!!  I had this in my develop branch - but this is a very subtle failure in the original code.

**Review by growlf — APPROVED** [review]
Well done - thank you for your improvements!

**Pull request merged**